users/views.py

```diff
@@ -66,3 +67,24 @@ def post(self, request):
         except User.DoesNotExist:
             return JsonResponse({'message' : 'INVALID_USER'}, status=401)
 
+class MemberShipOutView(View):
```

Comment:
회원 탈퇴라는 것이 결국에는 user의 정보를 삭제하는 것이므로 User view 안에 delete method로 구현 하면 좋을 것 같습니다.
추후 회원 정보를 주는 api를 추가 한다면 UserView에 get method로서 추가하면 될 것 같아요.
```suggestion
class UserView(View):
```